src/Form/Frontend/AbstractFrontendFormType.php

```diff
@@ -28,7 +35,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         parent::buildForm($builder, $options);
 
-        if (!isset($options['recaptcha']) || true === $options['recaptcha']) {
+        if ($this->recaptchaEnabled && (!isset($options['recaptcha']) || true === $options['recaptcha'])) {
```

Comment:
At a global level, we're checking the `RECAPTCHA_ENABLED` flag as an override, but I've retained the ability to turn on and off for individual forms.